novice-goals.Rmd

```diff
@@ -0,0 +1,210 @@
+# Outline {#outline}
+
+This outline describes the questions that the novice courses on R and Python will answer.
+The advanced course can then assume that learners have hands-on experience with these topics
+but nothing more.
+
+## Data manipulation
+
+-   How can I read tabular data into a program?
+    -   CSV
+    -   reading files
+-   How can I select subsets of my data?
+    -   select
+    -   filter
+    -   Boolean conditions
```

Comment:
Does arrange fit in this section somewhere?

Reply:
good catch - thanks